code/Player/HandViewModel.cs

```diff
@@ -39,7 +39,7 @@ public void SetHoldType(HoldType holdType)
             SetAnimInt(AnimKeyHoldType, (int) holdType);
             offset = holdType switch
             {
-                HoldType.None => NoneOffset,
+                HoldType.None => offset,
```

Comment:
pascalheu case

Reply:
Where ?

Reply:
The "offset" line 42 is to keep the same value, I prefer to let all possible values of the switch even if this case is doing nothing. It allows to throw exception in the default case